resolwe/flow/models/entity.py

```diff
@@ -48,7 +49,7 @@ def __str__(self):
 class Relation(BaseModel):
     """Relations between entities."""
 
-    class Meta(BaseCollection.Meta):
+    class Meta(BaseModel.Meta):
```

Comment:
[Django coding style guidelines](https://docs.djangoproject.com/en/2.0/internals/contributing/writing-code/coding-style/#model-style) suggest that `Meta` class should be placed after field names definitions. I know this is inconsistent with how things currently are in this project, but still, we may want to gradually move in that direction...